Formula/k/kubecolor.rb

```diff
@@ -0,0 +1,19 @@
+class Kubecolor < Formula
+  desc "Colorize your kubectl output"
+  homepage "https://kubecolor.github.io/"
+  url "https://github.com/kubecolor/kubecolor/archive/refs/tags/v0.2.2.tar.gz"
+  sha256 "ba0894a8e26fefff47a0691529964303bdd8fdc2d7ce74e7d241cb5a2f2ade50"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "kubernetes-cli" => :test
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    system bin/"kubecolor", "-h"
+    assert_equal 0, $CHILD_STATUS.exitstatus
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.